Analysis/src/TypeVar.cpp

```diff
@@ -1058,7 +1058,7 @@ ConstrainedTypeVarIterator end(const ConstrainedTypeVar* ctv)
 
 static std::vector<TypeId> parseFormatString(TypeChecker& typechecker, const char* data, size_t size)
 {
-    const char* options = "cdiouxXeEfgGqs";
+    const char* options = "cdiouxXeEfgGqs*";
```

Comment:
This will lead to number being inferred for `%*`

Reply:
... actually, not even that, since the scanning code skips until an alphabetic character. This should have a test confirming that `%*` can be used with any type from type checker perspective, see TypeInfer.builtins.test.cpp